generate_data.py

```diff
@@ -0,0 +1,117 @@
+from rapidsms.models import Contact, Connection, Backend
```

Comment:
How is this script used? Does it need to be run somewhere in the production environment or is it meant only for development or testing?

Reply:
Apologies, this was also accidently included in the submission